### DISCUSSION TITLE
Future dated posts appear instead of future:false is set

### DISCUSSION THREAD

**Discussion by FleXoft** [community/community#23548] [Other Feature Feedback, Questions, & Ideas]
<p>I’ve set future:false in my _config.yml and when run locally it does omit future dated posts. However, on github pages, it doesn’t seem to ignore future dated posts. It used to work 2-3 months ago…</p>

**Comment by yoannchaudet** [comment]
<p><a class="mention" href="/flexoft">@FleXoft</a> This should be fixed with <a href="https://github.com/actions/jekyll-build-pages/issues/9" class="inline-onebox">with future: false it is still showing all of my future blogs · Issue #9 · actions/jekyll-build-pages · GitHub</a> which was released a few minutes ago.</p>
<p>One gotcha, please note that if you care about the exact time a post is published, you will want to be explicit and define a <code>timezone</code> option in your Jekyll configuration file. We don’t guarantee which timezone the Actions runners are using.</p>

**Comment by FleXoft** [comment]
<p>We are talking days in this case, future posts are days, months in advance.</p>
